packages/bot-skeleton/.eslintrc.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
     },
     rules: {
         'no-underscore-dangle': 0,
-    }
+        'import/no-unresolved': 'off',
```

Comment:
😱 , is there any unresolved import in the site, i assume that would return an error

Reply:
it turned out that we don't need this configuration at all
It was because i was trying to run eslint tests before bootstrap and in that case it throws that error.
So the configurations removed